{
  "imports": {
    "class": "imports",
    "BroadworksFault": [
      "alarmName",
      "timeStamp",
      "subcomponent",
      "severity",
      "identifier",
      "systemName",
      "faultFields",
      "recommendedActionsText",
      "problemText",
      "component",
      "common",
      "alarmState"
    ],
    "SNMPv2-CONF": [
      "NOTIFICATION-GROUP",
      "MODULE-COMPLIANCE"
    ],
    "SNMPv2-SMI": [
      "MODULE-IDENTITY",
      "Unsigned32",
      "IpAddress",
      "Counter32",
      "Counter64",
      "Bits",
      "Integer32",
      "OBJECT-TYPE",
      "MibIdentifier",
      "Gauge32",
      "OBJECT-IDENTITY",
      "iso",
      "TimeTicks",
      "NOTIFICATION-TYPE"
    ],
    "SNMPv2-TC": [
      "TEXTUAL-CONVENTION",
      "DisplayString"
    ]
  },
  "systemFaults": {
    "name": "systemFaults",
    "oid": "1.3.6.1.4.1.6431.1.1.1",
    "class": "moduleidentity",
    "revisions": [
      {
        "revision": "2000-09-19 14:31",
        "description": ""
      }
    ],
    "lastupdated": "200201220000Z",
    "organization": "Broadsoft, Inc",
    "contactinfo": "Broadsoft, Inc. 220 Perry Parkway Gaithersburg, MD 20877 301-977-9440",
    "description": "The defines the fault "
  },
  "meta": {
    "comments": [
      "ASN.1 source file://MIBS/asn1/BW-ClientManagementAccessServerFault.mib",
      "Produced by pysmi-0.3.4 at Tue Apr 20 18:30:58 2021",
      "On host samy-ThinkPad-T495s platform Linux version 5.8.0-50-generic by user root",
      "Using Python version 3.8.5 (default, Jan 27 2021, 15:41:15) "
    ],
    "module": "BW-ClientManagementAccessServerFault"
  }
}